pkg/volume/azure_file/azure_file.go

```diff
@@ -207,7 +209,8 @@ func (b *azureFileMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 	os.MkdirAll(dir, 0750)
```

Comment:
This was changed in the original, but not here. Why?

Reply:
@msau42 The fix should only contains "work within different cloud environments", should not have azure file permission change which will break some user scenarios.